### DISCUSSION TITLE
Two of my sites are suddenly hitting Serverless Function rate limits on Vercel

### DISCUSSION THREAD

**Discussion by silasabbott** [sveltejs/kit#4464] [Q&A]
![image](https://user-images.githubusercontent.com/8752644/160708779-aa1a957e-074b-4d29-be35-48b4bd140602.jpeg)

I'm getting errors on two of my sites periodically because SvelteKit's index.js SSR serverless function somehow hit Vercel's limit of 1000 concurrent executions. I haven't touched or updated either project in 2 months, and this issue started happening a week and a half ago. They are both extremely low traffic sites. When this happens, it happens to both of them within 2 hours of each other.

Any ideas on what's happening?

**Comment by faissalhajar** [comment]
Hey @silasabbott!

I encountered this issue a couple of days ago and could not solve it. Many of my SvelteKit apps suddenly started showing the 429: TOO_MANY_REQUESTS error. The Vercel logs did not help much on debugging the problem.

The issue can be reproduced by rapidly reloading a SvelteKit app.

I am running the latest version of SvelteKit (to this date). I also tried changing the SvelteKit adapter from @sveltejs/adapter-auto  to @sveltejs/adapter-vercel without success.

**Vercel Logs**
`[GET] /23:09:27:50Function Status:NoneEdge Status:429Duration:N/AInit Duration:N/AMemory Used:N/AID:gru1::8wc4t-1651111767636-90a68d85e7bcUser Agent:Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/100.0.4896.88 Safari/537.36FUNCTION_RATE_LIMIT: This Serverless Function was rate limited`

If anyone finds out how to fix this issue, please let us know in this discussion.

Thank you!

**Reply by tomekrozalski** [reply]
@faissalhajar did you have this issue later? I had it last week. Updated svelte/kit and adapter packages helped. Weird.
